client/public/index.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Dashboard</title>
+    <meta name="description" content="Dashboard for github pull-requests notifier">
+    <meta name="author" content="wentwrong">
```

Comment:
For clarity, it's better to add `<script>` tags with references to the `<head>` section only and use a very simple `<script>` tag with inline code that will construct our client-side App by creating an app instance or calling a function.